### DISCUSSION TITLE
is there a way or an example how to display or integrate html reporting in azure devops?

### DISCUSSION THREAD

**Discussion by zbrea** [getgauge/gauge#2111] [General]
is there a way or an example how to display or integrate html reporting in azure devops?

**Comment by sriv** [comment]
there's some reference for setting up a gauge pipeline in azure - https://docs.gauge.org/howto/ci_cd/azure.html

As for html reporting in Azure - the last I checked there wasn't support in Azure, not sure if they've added support now.
